tests/test_coverage_boost.py

```
@@ -1,499 +1,292 @@
+        def mock_btn2(*args, **kwargs):
+            nonlocal del_cb_prop
+            if kwargs.get('icon') == 'delete_outline':
+                del_cb_prop = kwargs.get('on_click')
+            return MagicMock()
+        mock_ui.button.side_effect = mock_btn2
+
+        ui_inst.draw_editor("root/list")
+        if del_cb_prop:
+             try:
+                 del_cb_prop()
+             except: pass
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)